serve/httprule/handler.go

```diff
@@ -73,6 +77,18 @@ func WithRuleTemplates(httpRuleTemplates []*annotations.HttpRule) Option {
 	}
 }
 
+// WithDefaultHandler is an [Option] to configure a [Handler] with a fallback
+// handler when the request being handled does not match any of the gRPC
+// methods the [Handler] is configured with. By default the [Handler] will
+// return a 404 NotFound response. If a default handler is supplied, it will be
+// called instead of returning that 404 NotFound response.
+func WithDefaultHandler(next http.Handler) Option {
```

Comment:
nit pickery, but why call this parameter `next`? Isn't that usually used for middleware?

Reply:
I originally called this `WithNextHandler` because it is kind of a middleware-like change anyway. But I renamed the terminology to "DefaultHandler", but I could not rename this parameter to `default` as that's a keyword, so I just left it as `next`